datacommons/examples/simple_query.py

```diff
@@ -45,7 +45,10 @@ def main():
   with pd.option_context('display.width', 400, 'display.max_rows', 100):
     print(df)
 
-  dc.save_dataframe(df, 'simple_query_result')
+  dc.save_dataframe(df, 'test_df')
+  saved_pd = dc.read_dataframe('test_df')
+  with pd.option_context('display.width', 400, 'display.max_rows', 100):
+    print(saved_pd)
```

Comment:
Can you compare the original and saved/restored dataframes?

Reply:
Added comparison